src/command_modules/azure-cli-role/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.19
 ++++++
+* role assignments: expose "role assignment list-changelogs" for rbac audit 
 * Minor fixes.
```

Comment:
Minor fixes should only appear if it is the only thing in history. Since you have something substantial you can remove it.